include/swift/IDE/IDEBridging.h

```diff
@@ -102,29 +92,40 @@ ResolvedLocVector ResolvedLocVector_createEmpty();
 ///   In that case `swift_SwiftIDEUtilsBridging_runNameMatcher` can return a
 ///   `ResolvedLocVector`.
 class BridgedResolvedLocVector {
-  const std::vector<ResolvedLoc> *vector;
+  friend void *BridgedResolvedLocVector_getOpaqueValue(const BridgedResolvedLocVector &vector);
+
+  std::vector<ResolvedLoc> *vector;
 
 public:
   /// Create heap-allocaed vector with the same elements as `vector`.
   BridgedResolvedLocVector(const std::vector<ResolvedLoc> &vector);
 
   /// Create a `BridgedResolvedLocVector` from an opaque value obtained from
   /// `getOpaqueValue`.
-  BridgedResolvedLocVector(const void *opaqueValue);
+  BridgedResolvedLocVector(void *opaqueValue);
+
+  void push_back(const ResolvedLoc &Loc);
 
   /// Get the underlying vector.
   const std::vector<ResolvedLoc> &unbridged();
 
   /// Delete the heap-allocated memory owned by this object. Accessing
   /// `unbridged` is illegal after calling `destroy`.
   void destroy();
-
-  /// Get an opaque pointer value that describes this 
-  /// `BridgedResolvedLocVector`. This opaque value can be returned by a
-  /// `@_cdecl` function in Swift.
-  const void *getOpaqueValue() const;
 };
 
+
+SWIFT_NAME("BridgedResolvedLocVector.empty()")
+BridgedResolvedLocVector BridgedResolvedLocVector_createEmpty();
+
+/// Get an opaque pointer value that describes this
+/// `BridgedResolvedLocVector`. This opaque value can be returned by a
+/// `@_cdecl` function in Swift.
+/// 
+/// - Note: Cannot be defined as a member on BridgedResolvecLocVector because 
+///   Swift 5.8 does not import methods that return pointers. 
```

Comment:
You can add `SWIFT_IMPORT_UNSAFE` to allow pointer imports in 5.8

Reply:
Oh nice. I didn’t know about that.